src/webui/api/torrentscontroller.h

```diff
@@ -67,6 +67,7 @@ private slots:
     void addTrackersAction();
     void editTrackerAction();
     void removeTrackersAction();
+    void removeTrackerAction();
```

Comment:
Looking at the names of these two endpoints, one have a false assumption about their purpose (the first one removes several trackers, and the second one removes one). And knowing exactly what they do, I would not call these names appropriate (namely the second endpoint name, since the previous one already existed).

Reply:
What would you name it instead?

Reply:
Given that there is already an endpoint with a suitable name (aka `removeTrackers`), but with a narrower scope, I would just generalize it to serve this use case as well. I.e. if there is no `hash` parameter provided (or if it is empty), then `removeTrackers` could remove the trackers from all torrents.

Reply:
Sure, will do.